### DISCUSSION TITLE
Ragarding Backup

### DISCUSSION THREAD

**Discussion by Farashp7** [wazuh/wazuh#17293] [Q&A] [answered]
Hai team,
For how many days will be logs avilable in wazuh server?
If I need to take backups of the logs, Which all files should be backed up.
Is there any inbuilt technology in wazuh to take backup to a remote server?

Thanks in advance

**Comment by davidjiglesias** [comment, marked as answer]
Already answered and being treated in https://wazuh.slack.com/archives/C0A933R8E/p1685342972990039
